gradle-sls-packaging/src/test/groovy/com/palantir/gradle/dist/tasks/CreateManifestTaskIntegrationSpec.groovy

```diff
@@ -442,6 +442,16 @@ class CreateManifestTaskIntegrationSpec extends GradleIntegrationSpec {
         file('bar-server/product-dependencies.lock').readLines().contains 'com.palantir.group:foo-service (0.0.0, 1.x.x)'
     }
 
+    def "check depends on createManifest"() {
+        file("product-dependencies.lock").delete()
```

Comment:
why?

Reply:
so the task doesn't fail

Reply:
yep setup() was broken before - fixed as part of #419  so it shouldn't be necessary anymore